send-data/aws-lambda.mdx

```diff
@@ -199,3 +199,15 @@ To turn off Amazon CloudWatch logging, deny the Lambda function access to Amazon
 - Check the function logs on the AWS console. The Axiom Lambda Extension logs any errors with setup or ingest.
 
 For testing purposes, set the `PANIC_ON_API_ERR` environment variable to `true`. This means that the Axiom Lambda Extension crashes if it can’t connect to Axiom.
+
+## Best practices using AWS Lambda
+
+When using AWS Lambda for ingestion, consider these additional recommendations:
+
+- Axiom Lambda extension: Use the [Axiom Lambda extension](/docs/send-data/aws-lambda) for efficient log collection directly from your Lambda functions.
+
+- Minimize direct API calls: Avoid making individual API calls for each Lambda invocation. Instead, use the [Lambda extension](/docs/send-data/aws-lambda) or aggregate logs before sending.
```

Comment:
These feel like the same thing really, both explanations are good and valid, but ultimately are the same right?